src/views/Entries/QualityControl/EntryCard/index.tsx

```diff
@@ -168,6 +209,27 @@ function EntryCard(props: EntryCardProps) {
                             }
                         />
                     )}
+                    <Cloak
+                        hide={shouldHideLeadEdit}
+                        render={
+                            <WarningButton
```

Comment:
Let's use modalize instead of handling it separately if possible.

Reply:
Is it because lead information is incomplete?

Reply:
it is because I had to send out the request on click